common/models/report-model.js

```diff
@@ -5,9 +5,10 @@ var request = require('request-promise');
 var _ = require('underscore');
 
 var path = require('path');
+var appRoot = require('app-root-path');
+var modulePath = appRoot.require('/node_modules/loopback-connector-mongodb/node_modules/mongodb');
```

Comment:
I thought that the docker deployment placed `node_modules` at the same level as appRoot and not under it?

```
# I expected that before docker we had:
- appRoot
   - node_modules
   - etc

# we changed this for optimization of docker image builds like
- appRoot
   - etc
- node_modules

# ... so we needed to change the code too
# but now I see the `app-root-path` code that seems to be written again for:
- appRoot
   - node_modules
   - etc
```